war/Mat.css

```diff
@@ -2956,3 +2958,11 @@ table#MostRecentActivityCellTable {
 .table td {
     vertical-align: middle;
 }
+
+.action-button-bar {
```

Comment:
Do we need this blank style?

Reply:
Not that we really need it. I was thinking to keep an empty style as a placeholder for a common style for all instances of toolbars. Do you think we don't need it?

Reply:
In addition to that, the style is used as a classifier for the buttons. So all toolbar buttons have same padding.